### PR TITLE
set alias default translation strategy

### PR DESCRIPTION
**Pull request by andrey1s**

### CODE REVIEW

**Comment by OskarStark** [comment]
ping @rande

**Comment by greg0ire** [comment]
Would fix #2419 very simply :) :+1:

**Comment by OskarStark** [comment]
@andrey1s i would like to merge your PR, but could you please add a test that the configuration is processed correctly?

Thank you!

**Comment by andrey1s** [comment]
@OskarStark I don't use the Sonata configuration, only the configuration of the Symfony 2. and create alias for the default strategy. I think the configuration for the Sonata this could be another PR if needed.

**Comment by greg0ire** [comment]
There is a problem with this though : you cannot associate a strategy with a given bundle. So, if you're using SonataUserBundle, changing the strategy will break all the translations and you will have to translate everything again. See #3319

**Comment by andrey1s** [comment]
@greg0ire this PR not changing the default strategy. And of course if the developer changes the strategy then of course he needs to translate.

**Comment by greg0ire** [comment]
> if the developer changes the strategy then of course he needs to translate

Yes, it's the problem I'm warning you about. It would be better to be able to change the default strategy for a set of bundles only.

**Comment by greg0ire** [comment]
But maybe this could be merged anyway, with a warning about this problem in the docs? It would be useful for people who only use the Admin Bundle I guess…

File: Resources/doc/reference/translation.rst [hunk 1]
```diff
@@ -151,6 +151,16 @@ apps based on the field name. It is reasonable to start with the ``native`` stra
...
 
+You can replace default strategy in ``config.yml``:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        services:
+            sonata.admin.label.strategy.default:
+                alias: sonata.admin.label.strategy.underscore
```
Comment (core23) [inline Resources/doc/reference/translation.rst]: Please add a warning that was mentioned by @greg0ire

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**